test/core/internals.js

```diff
@@ -105,6 +105,16 @@ describe('Core htmx internals Tests', function() {
     htmx._('shouldCancel')({ type: 'click' }, button).should.equal(true)
   })
 
+  it('tags respond correctly to shouldCancel even if they are outside of a form', function() {
+    var form = make("<form id='f1'></form><input id='i1' form='f1' type='submit'>")
+    var input = byId('i1')
+    htmx._('shouldCancel')({ type: 'click' }, input).should.equal(true)
+
+    form = make("<form id='f1'></form><button id='b1' form='f1' type='submit'>")
+    var button = byId('b1')
+    htmx._('shouldCancel')({ type: 'click' }, button).should.equal(true)
+  })
+
```

Comment:
I think if you put an hx-get on the form this could be an end to end test, rather than needing to reach into the internals of htmx.  Does that sound reasonable to you?